pages/common/logstash.md

```diff
@@ -0,0 +1,21 @@
+# logstash
+
+> An ETL tool, usually used to ingest data from various sources, like databases and flat log files into elasticsearch.
```

Comment:
Please expand the acronym within brackets. So that newbies don't have a hard time :)